weave/trace_server/opentelemetry/python_spans.py

```diff
@@ -287,8 +287,27 @@ def to_call(
     ) -> tuple[tsi.StartedCallSchemaForInsert, tsi.EndedCallSchemaForInsert]:
         events = [SpanEvent(e.as_dict()) for e in self.events]
         usage = get_weave_usage(self.attributes) or {}
+
         inputs = get_weave_inputs(events, self.attributes) or {}
+        # Only de-nest if we have one key
+        if len(inputs) == 1:
```

Comment:
I'm open to discussing this but I added this check for if there's multiple keys. It feels like we shouldn't just unpack and drop the other keys.